ads/insticator.js

```diff
@@ -0,0 +1,91 @@
+// !!!!!!!!!! ---------- AMP EMBED - INSTICATOR ---------- !!!!!!!!!! //
+
+import { validateData, loadScript } from '../3p/3p';
+
+// ------- HELPER VARIABLES ------- //
+//reusable URL references to ads, embed and the library
+//Don't forget to preconnect and prefetch as it's described in the docs
+//https://github.com/ampproject/amphtml/blob/master/ads/README.md
+const url = {
+  ads: 'https://drhn9v8cwg89y.cloudfront.net',
+  embed: 'https://d3lcz8vpax4lo2.cloudfront.net',
+  lib: 'https://testthisshit.online/amp-embed-lib/insticator.js'
+};
+
+// ------- HELPER FUNCTIONS ------- //
+function createInitialMarkup() {
```

Comment:
for flexibility in a long run, I suggest you only leave config related logic in this file, and move logic to `/amp-embed-lib/insticator.js` as much as possible.